pallets/randomness/Cargo.toml

```diff
@@ -19,34 +19,26 @@ sp-core = { workspace = true }
 sp-io = { workspace = true }
 sp-runtime = { workspace = true }
 sp-std = { workspace = true }
-sp-consensus-babe = { workspace = true }
 codec = { workspace = true }
 
-# Benchmarks
```

Comment:
Isn't this a false positive too? Shouldn't this be enabled when building with `runtime-benchmarks`?

Reply:
The problem with `runtime-benchmarks` feature is that is wasn't working even before this, so I don't know. I tried to fix it yesterday but honestly couldn't.